package.json

```diff
@@ -26,6 +28,19 @@
     "chokidar": "^3.0.2"
   },
   "devDependencies": {
-    "standard": "^13.1.0"
+    "@typescript-eslint/eslint-plugin": "^2.23.0",
+    "@typescript-eslint/parser": "^2.23.0",
+    "electron": "^8.1.1",
+    "eslint": "^6.8.0",
+    "eslint-config-standard-with-typescript": "^14.0.0",
+    "eslint-plugin-import": "^2.20.1",
+    "eslint-plugin-node": "^11.0.0",
+    "eslint-plugin-promise": "^4.2.1",
```

Comment:
I have (successfully) avoided using eslint explicitly up to now; are all these plug-ins necessary?

Reply:
Fixed, check latest. Removed `eslint` in favor of [ts-standard](https://github.com/toddbluhm/ts-standard)